tests/assert.rs

```diff
@@ -0,0 +1,147 @@
+use futures::TryStreamExt;
+use k8s_openapi::api::core::v1::Pod;
+use kube::api::{Api, LogParams};
+
+pub async fn pod_log_equals(
+    pods: &Api<Pod>,
+    pod_name: &str,
+    expected_log: &str,
+) -> anyhow::Result<()> {
+    let mut logs = pods.log_stream(pod_name, &LogParams::default()).await?;
+
+    while let Some(chunk) = logs.try_next().await? {
+        assert_eq!(expected_log, String::from_utf8_lossy(&chunk));
+    }
+
+    Ok(())
+}
+
+pub async fn pod_log_contains(
+    pods: &Api<Pod>,
+    pod_name: &str,
+    expected_log: &str,
+) -> anyhow::Result<()> {
+    let logs = pods.logs(pod_name, &LogParams::default()).await?;
+    assert!(
+        logs.contains(expected_log),
+        format!("Expected log containing {} but got {}", expected_log, logs)
+    );
+    Ok(())
+}
+
+pub async fn pod_container_log_contains(
+    pods: &Api<Pod>,
+    pod_name: &str,
+    container_name: &str,
+    expected_log: &str,
+) -> anyhow::Result<()> {
+    let mut log_params = LogParams::default();
+    log_params.container = Some(container_name.to_owned());
+    let logs = pods.logs(pod_name, &log_params).await?;
+    assert!(
+        logs.contains(expected_log),
+        format!("Expected log containing {} but got {}", expected_log, logs)
+    );
+    Ok(())
+}
+
+// pub async fn pod_log_does_not_contain(
```

Comment:
was this intended to be left in, or should we remove this?

Reply:
It's from a test that I was meaning to implement for the init containers work but didn't get straight in my mind.  I would kind of like to come back to it so hadn't purged it, but it would be no loss to get rid of it; it wasn't quite right for what I wanted, and would be easy to reconstruct.